assets/src/block-templates/template-list.js

```diff
@@ -5,6 +5,7 @@ import * as realityCheck from './reality-check';
 import * as issues from './issues';
 import * as pageHeader from './page-header';
 import * as highlightedCta from './highlighted-cta';
+import * as homepage from './homepage';
```

Comment:
Just an idea but maybe we could rename the `homepage` folder to `homepage-layout`, to differentiate between the layouts and other templates? 🤔